app/src/main/admin/components/ModellingGroups/ListOfUsers.tsx

```diff
@@ -24,5 +25,14 @@ export class ListOfUsers extends React.Component<Props, undefined> {
                                   showDelete={isAdmin}
             />;
         });
+
+    }
+}
+
+const mapStateToProps = (state: any) => {
+    return {
+        isAdmin: state.auth.permissions.indexOf("*/modelling-groups.manage-members") > -1 ? true: false,
```

Comment:
Feels so much cleaner having this logic here :)
Why do we need the ternary operator here? Isn't
`state.auth.permissions.indexOf("*/modelling-groups.manage-members") > -1 ? true: false`
equivalent to
`state.auth.permissions.indexOf("*/modelling-groups.manage-members") > -1` ?